addons/point_of_sale/static/src/js/Screens/PaymentScreen/PaymentScreen.js

```diff
@@ -228,6 +228,7 @@ odoo.define('point_of_sale.PaymentScreen', function (require) {
                     syncedOrderBackendIds = await this.env.pos.push_single_order(this.currentOrder);
                 }
             } catch (error) {
+                this.env.services.ui.unblock()
```

Comment:
i don't get this. Isn't there already an `unblock` in the `finally` clause?

Reply:
@vlst-odoo The issue arises when an error occurs, causing a popup to appear before unblocking the UI. Consequently, it becomes impossible to close the popup, preventing the execution of the code within the `finally` block.

Reply:
I see. Can you please put this info in a comment?

Reply:
@vlst-odoo Thanks for your review. I added a comment.